examples/nextjs-zip/app/Makefile

```diff
@@ -0,0 +1,16 @@
+install:
+	npm install
+
+build:
+	npm run build
+
+artifacts:
+	# Static resources ideally served from CloudFront
+	cp -r public/ .next/standalone/public
+	cp -r .next/static/ .next/standalone/.next/static
+	
+	# Copy artifacts for deployment
+	cp -r .next/standalone/ $(ARTIFACTS_DIR)
```

Comment:
Change this line to this: `cp -r .next/standalone/. $(ARTIFACTS_DIR)`

This works on both LInux and Macos.

Reply:
Please also look at other copy commands in the Makefile.